content/en/blog/2021/wasm-api-alpha/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Announcing the alpha availability of WebAssembly Plugins
+description: Introduction of Wasm Plugin API, and updates on Envoy and Istio's Wasm based plugins.
+publishdate: 2021-11-25
+attribution: "Daniel Grimm(Red Hat), Pengyuan Bian(Google), Takeshi Yoneda(Tetrate)"
+
+keywords: [wasm,extensibility,WebAssembly]
+---
+
+[At the Istio 1.9 release](../wasm-progress/), we introduced experimental support for Wasm module distribution and Wasm extensions ecosystem repository for canonical examples and use cases of Wasm extension development. Over the past 9 months, the Istio, Envoy, and Proxy-Wasm communities have continued our joint efforts to make WebAssembly (Wasm) extensibility stable, reliable, and easy to adopt, and we are pleased to announce Alpha support for Wasm extensibility at Istio 1.12 release! Let’s walk through the updates to Wasm support through the Istio 1.12 release.
```

Comment:
In the Istio 1.9...    extensibility in the Istio 1.12

The first sentence doesn't make sense to me. To many and's? 

Also make the WebAssembly (Wasm) in the first sentence.

Reply:
@bianpengyuan wdyt? ^^